### DISCUSSION TITLE
Permissions are not working

### DISCUSSION THREAD

**Discussion by CactusKipic** [requarks/wiki#3415] [Help / Questions]
**Question**
I'm trying to set up permission on my wiki.js but no matter which persmission is given to a group, the users belonging to it, which are registered and validated, are still not able to create any page, they are only able to read pages. It is possible to use the 'new page' button, but after selecting the location of the page and going and the next step, the wiki prompt an 'unauthorized' page saying the user can't create a page.
Plus, since it seems impossible to put a user in the group administrator, I've made a group with all permissions checked, and this doesn't allow to do anything, even checking the administration panel, which is accessible to everyone, say "not authorized" when accessed.
I'm using Discord for user connection, I don't think the problem could come from here since I did not have any problem setting the identification, and user are correctly verified and put in the right group when  they registered.

Something even more strange appear juste before I was finishing redacting this post, as I was trying a last time to create a page with a user other than the root user, it worked for no apparent reason plus I did not have access to the admin panel. I tried to change the user to the admin group I created to see if permissions were magically working now, I was able to take access to the admin panel through the home page, but got the same message saying that I could not access the panel (last image in the album). And since then, the problem is back exactly as it were in the beginning.

[Here is an album](https://imgur.com/a/3OvhfBH) showing the permissions settings used and the error page.

**Host Info:**
OS: Debian (buster) 10 x64
Wiki.js version: 2.4.107
Database engine: postgres 11.7

**Comment by NGPixel** [comment]
Did you create a [page rule](https://docs.requarks.io/groups)? Global permissions by themselves don't do anything.

**Comment by Jhobean** [comment]
I had the same bug. What I did is modify something in each usergroup and save. Seem the save of user group fixed it.

**Comment by CactusKipic** [comment]
So in the end, it looks like, as you said, Global permissions doesn't do anything. It was actually working for classic user so I didn't touch it before having time, and possibility to test with someone. And it happens that permissions are weirdly updated, just reloading a page doesn't work, even witch cache reloading, but make it work by changing page (ex: going on the profile).
So I gave all permission for the custom administrator and yeah, it allows to do everything, except for the administration panel, where there is still the message saying that I'm not allowed to access this resource, but can still access it. And from what I can see and do, the permissions work, but aren't 'complete' as the panel can't display the number of pages, groups, and user, even though I can see them when looking in the right tab.
![firefox_2020-07-30_02-00-46](https://user-images.githubusercontent.com/13261381/88865553-7d2e9e80-d208-11ea-89c4-85f49ce0a5ef.png)

**Reply by Stack-Baize** [reply]
I have the same problem, updated to the latest version and still the same. I feel that there is a problem with the custom setting permissions, and I will prompt when I open the management console, but the permission settings can be operated normally.

**Reply by jsprds** [reply]
Hi @CactusKipic and @Stack-Baize , did you find a solution for this problem? I'm facing the same issue. Thanks!

**Comment by Jhobean** [comment]
You try a wiki reboot?

**Comment by CactusKipic** [comment]
I just did, and it didn't change anything. But in the end, it's just "visual" so as long as the permissions to edit groups and players are working, that' okay.

**Comment by col-panic** [comment]
I use 2.5.170 - and I keep reading about the global permissions, on the Page Rules tab it says to **enable Global Permissions** but I simply can't find this preference option anywhere?! Where did you guys enable those global permissions?

![grafik](https://user-images.githubusercontent.com/1679857/100740108-72fbf700-33d8-11eb-867a-dc8251b6556a.png)

**Comment by col-panic** [comment]
oh ok, it seems like this Info note means something like: **It is required to have a permission globally enabled (within the permissions tab) in order for it to be applicable on a page rule**
